js/views/inbox_view.js

```diff
@@ -38,16 +38,25 @@
     Whisper.ConversationStack = Whisper.View.extend({
         className: 'conversation-stack',
         open: function(conversation) {
-            var $el = this.$('#conversation-' + conversation.cid);
-            if ($el === null || $el.length === 0) {
-                var view = new Whisper.ConversationView({
-                    model: conversation,
-                    appWindow: this.model.appWindow
+            var id = 'conversation-' + conversation.cid;
+            if(id !== this.el.firstChild.id) {
+                this.$("video").each(function() {
```

Comment:
Could we limit this to audio/video tags within the firstChild to make the query lighter?